src/PluginsManager/Microsoft.Performance.Toolkit.PluginsManager.Core/AvailablePlugin.cs

```diff
@@ -56,8 +58,15 @@ public sealed class AvailablePlugin
         public Uri PluginPackageUri { get; }
 
         /// <summary>
-        ///     Gets the Guid which identifies the platform (NuGet, Github etc.) the plugin package is hosted on.
+        ///     Gets the Guid which identifies the unique <see cref="Transport.IPluginFetcher"/> resouce
+        ///     the plugin package should be fetched from.
         /// </summary>
-        public Guid FetcherTypeId { get; }
+        public Guid FetcherResourceId { get; }
```

Comment:
This feels silly... you'd need to fetch the available plugin to know what this value should be. If you fetched it already, why do you need this value?

Reply:
Not quite sure what you mean, but my understanding of fetching is getting the package content.  AvailablePlugin represents a discovered plugin whose content is ready to be fetched. It needs the fetcher id to know which fetcher resource it should use to fetch the package content.

Reply:
Resolved offline